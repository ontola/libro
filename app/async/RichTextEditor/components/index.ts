export * from './EditableWithPlugins';
export * from './ElementButton';
export * from './InputDialog';
export * from './ListButton';
export * from './MarkButton';
export * from './RichTextEditor';
export * from './RichTextEditorMd';
export * from './ToggleButton';
export * from './ToggleButtonWithInputDialog';
