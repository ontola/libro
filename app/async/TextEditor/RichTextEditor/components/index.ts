export * from './EditableWithPlugins';
export * from './RichTextEditor';
export * from './RichTextEditorMd';
