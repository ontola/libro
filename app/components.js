export { default as ActionsBar } from './components/ActionsBar/index';
// export { default as Attachment } from './components/Attachment/index';
export { default as Attachment } from './components/Attachment/index';
export { default as BackButton } from './components/BackButton/index';
export { default as BottomBar } from './components/BottomBar/index';
export { default as BottomBarButton } from './components/BottomBarButton/index';
export { default as Button } from './components/Button/index';
export { default as BlurButton } from './components/BlurButton/index';
export { default as Card } from './components/Card/index';
export { default as CardActions } from './components/Card/CardActions';
export { default as CardAppendix } from './components/Card/CardAppendix';
export { default as CardButton } from './components/Card/CardButton';
export { default as CardContent } from './components/Card/CardContent';
export { default as CardDivider } from './components/Card/CardDivider';
export { default as CardFixed } from './components/Card/CardFixed';
export { default as CardLink } from './components/Card/CardLink';
export { default as CardMain } from './components/Card/CardMain';
export { default as CardMenuFloater } from './components/Card/CardMenuFloater';
export { default as CardRow } from './components/Card/CardRow';
export { default as CardMicroRow } from './components/Card/CardMicroRow';
export { default as Carousel } from './components/Grid/Carousel';
// export { default as ChronoFeed } from './components/ChronoFeed/index';
// export { default as ChronoItem } from './components/ChronoItem/index';
export { default as Collapsible } from './components/Collapsible/index';
export { default as CollapseText } from './components/CollapseText/index';
export { default as Column } from './components/Column/index';
export { default as Columns } from './components/Columns/index';
// export { default as CompareVotesBar } from './components/CompareVotesBar/index';
export { default as Container } from './components/Container/index';
export { default as CoverImage } from './components/CoverImage/index';
// export { default as CountBubble } from './components/CountBubble/index';
export { default as Cover } from './components/Cover/index';
export { default as Detail } from './components/Detail/index';
export { default as DetailText } from './components/Detail/text';
export { default as DetailImage } from './components/Detail/image';
export { default as DetailDate } from './components/DetailDate/index';
// export { default as DetailDuration } from './components/DetailDuration/index';
export { default as DetailsBar } from './components/DetailsBar/index';
export { default as DetailStatus } from './components/DetailStatus/index';
export { default as DetailType } from './components/DetailType/index';
export { default as DetailVotedFor } from './components/DetailVotedFor/index';
// export { default as DonutChart } from './components/DonutChart/index';
export { default as Drawer } from './components/Drawer/index';
export { default as Dropdown } from './components/Dropdown/index';
export { default as DropdownLink } from './components/Dropdown/DropdownLink';
export { default as Error } from './components/Error/index';
// export { default as EventItemShow } from './components/EventItemShow/index';
export {
  Form,
  FormFooter,
  FormSection,
} from './components/Form/index';
export { default as FormField } from './components/FormField/index';
export { default as Grid } from './components/Grid/index';
export { default as GeneratedIcon } from './components/GeneratedIcon/index';
export { default as Header } from './components/Header/index';
export { default as Heading } from './components/Heading/index';
export { default as HiddenFormField } from './components/HiddenFormField/index';
export { default as hoverBox } from './components/HoverBox/index';
export { default as Image } from './components/Image/index';
export {
  Input,
  FileInput,
} from './components/Input/index';
export { default as LabelValue } from './components/LabelValue/index';
export { default as LabelValueBar } from './components/LabelValueBar/index';
export { default as LDLink } from './components/LDLink/index';
// export { default as LinkedDetailDate } from './components/LinkedDetailDate/index';
export { default as HoverPopup } from './components/LinkedPopup/HoverPopup';
export { default as LinkedPopup } from './components/LinkedPopup/index';
export { default as LinkCard } from './components/LinkCard/index';
export { default as LinkDuo } from './components/LinkDuo/index';
export { default as LinkList } from './components/LinkList/index';
export { default as List } from './components/List/index';
export {
  default as Loading,
  LoadingCard,
  LoadingCardFixed,
  LoadingMicroRow,
  LoadingDetail,
  LoadingPage,
  LoadingParent,
  LoadingRow,
  LoadingSideBarLink,
} from './components/Loading/index';
export { default as Markdown } from './components/Markdown/index';
export { default as MarkdownFixedPreview } from './components/MarkdownFixedPreview/index';
export { default as NavBarContent } from './components/NavBarContent/index';
export { default as Notification } from './components/Notification/index';
export { default as Omniform } from './components/Omniform/index';
export { default as OmniformFields } from './components/Omniform/OmniformFields';
export { default as OmniformPreview } from './components/Omniform/OmniformPreview';
export { default as OrgSwitcher } from './components/OrgSwitcher/index';
export { default as PageHeader } from './components/PageHeader/index';
export { default as ProfileCard } from './components/ProfileCard/index';
export { default as ProfileListItem } from './components/ProfileListItem/index';
export { default as Progress } from './components/Progress/index';
export { default as ProgressBar } from './components/ProgressBar/index';
// export { default as SearchResultItem } from './components/SearchResultItem/index';
export { default as Resource } from './components/Resource/index';
export { default as SideBar } from './components/SideBar/index';
export { default as SideBarLink } from './components/SideBarLink/index';
export { default as SideBarCollapsible } from './components/SideBarCollapsible/index';
export {
  SignInFormCard,
  SignInFormCardRow,
} from './components/SignInForm/index';
export { default as SkipNavigation } from './components/SkipNavigation/index';
export { default as Spinner } from './components/Spinner/index';
export { default as TabBar } from './components/TabBar/index';
export { default as Tab } from './components/TabBar/Tab';
export { default as Tag } from './components/Tag/index';
export { default as TextEditor } from './components/TextEditor/index';
// export { default as VideoPlayer } from './components/VideoPlayer/index';
// export { default as VoteChart } from './components/VoteChart/index';
// export { default as VoteData } from './components/VoteData/index';
export { default as WidgetTopology } from './components/Widget/WidgetTopology';
