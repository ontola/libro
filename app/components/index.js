// @flow
export { default as App } from './App';
export { default as ArgumentShow } from './ArgumentShow';
export { default as ArgumentListItem } from './ArgumentListItem';
export { default as Box } from './Box';
export { default as Button } from './Button';
export { default as Columns } from './Columns';
export { default as Container } from './Container';
export { default as Cover } from './Cover';
export { default as DetailProfile } from './DetailProfile';
export { default as DetailStatus } from './DetailStatus';
export { default as DetailType } from './DetailType';
export { default as Detail } from './Detail';
export { default as DetailsBar } from './DetailsBar';
export { default as DocumentShow } from './DocumentShow';
export { default as Drawer } from './Drawer';
export { default as EntryListItem } from './EntryListItem';
export { default as Heading } from './Heading';
export { default as HoverBox } from './HoverBox';
export { default as HitStats } from './HitStats';
export { default as LinkList } from './LinkList';
export { default as List } from './List';
export { default as MotionShow } from './MotionShow';
export { default as MotionsListItem } from './MotionsListItem';
export { default as NavbarWrapper } from './NavbarWrapper';
export { default as Notification } from './Notification';
export { default as ProgressBar } from './ProgressBar';
export { default as ResetFiltersDisplay } from './ResetFiltersDisplay';
export { default as SearchPage } from './SearchPage';
export { default as Opinion } from './Opinion';
export { default as Opinions } from './Opinions';
export { default as PoliticiansListItem } from './PoliticiansListItem';
export { default as ProfileCard } from './ProfileCard';
export { default as VoteButtons } from './VoteButtons';
export { default as VoteChart } from './VoteChart';
export { default as VoteData } from './VoteData';

import * as Components from '.';
export default Components;
