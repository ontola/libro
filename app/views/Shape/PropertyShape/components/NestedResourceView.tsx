import { Node } from '@ontologies/core';
import sh from '@ontologies/shacl';
import { LinkedResourceContainer, Property, useLRS } from 'link-redux';
import React, { FC } from 'react';

export interface Props {
  targetValue: Array<{ '@id': Node }>;
  nestedShape: boolean;
  targetNode: Node;
  theme: string;
  onKeyUp: () => void;
  removeItem: () => void;
  shClass: Node;
}

export const NestedResourceView: FC<Props> = (props) => {
  const {
    targetValue,
    nestedShape,
    targetNode,
    theme,
    onKeyUp,
    shClass,
  } = props;
  const lrs = useLRS();

  if (typeof targetValue === 'undefined') {
    return null;
  }

  const targetShape = lrs.store.find(
    null,
    sh.targetClass,
    shClass,
    null,
  );

  const children = !targetShape
    ? (
      <Property
        label={sh.class}
        nestedShape={nestedShape}
        targetNode={targetNode}
        theme={theme}
        onKeyUp={onKeyUp}
        {...props}
      />
    )
    : (
      <LinkedResourceContainer
        subject={targetShape.subject}
        {...props}
      />
    );

  return (
    <React.Fragment>
      {children}
    </React.Fragment>
  );
};
