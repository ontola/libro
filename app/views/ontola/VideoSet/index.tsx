import {
  FC,
  register,
  useProperty,
} from 'link-redux';
import React, { VideoHTMLAttributes } from 'react';
import { FormattedMessage } from 'react-intl';

import ontola from '../../../ontology/ontola';
import { allTopologies } from '../../../topologies';

export interface VideoSetProps extends VideoHTMLAttributes<HTMLVideoElement> {
  className: string,
}

const VideoSet: FC<VideoSetProps> = (props) => {
  const [ariaLabel] = useProperty(ontola.ariaLabel);

  const [mp4] = useProperty(ontola['format/mp4']);
  const [mov] = useProperty(ontola['format/mov']);
  const [webm] = useProperty(ontola['format/webm']);

  const videoElement = React.useRef<HTMLVideoElement>(null);

  React.useEffect(() => {
    videoElement.current?.load();
  }, [webm, mp4, mov]);

  return (
    <video aria-label={ariaLabel?.value} {...props} ref={videoElement}>
      {webm && <source src={webm.value} type="video/webm" />}
      {mp4 && <source src={mp4.value} type="video/mp4" />}
      {mov && <source src={mp4.value} type="video/quicktime" />}


      <p>
        <FormattedMessage
          defaultMessage="No video support"
          id="https://ns.ontola.io/app/videoUnsupported"
        />
        <a href={mp4?.value ?? webm?.value}>click here to download</a>
      </p>
    </video>
  );
};

VideoSet.type = ontola.VideoSet;

VideoSet.topology = allTopologies;

export default [
  ...register(VideoSet),
];
