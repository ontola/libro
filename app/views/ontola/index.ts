import PictureSet from './PictureSet';
import VideoSet from './VideoSet';

export default [
  ...PictureSet,
  ...VideoSet,
];
