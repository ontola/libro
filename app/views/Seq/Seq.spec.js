import rdf, { createNS } from '@ontologies/core';
import * as rdfx from '@ontologies/rdf';
import * as schema from '@ontologies/schema';

import ontola from '../../ontology/ontola';

import components from './';

const resource = rdf.blankNode('g70120412320900');

const testNS = createNS('https://argu.dev/');

const seq = [
  testNS('menus/info#about'),
  testNS('menus/info#team'),
  testNS('menus/info#governments'),
  testNS('menus/info#press_media'),
  testNS('menus/info#support'),
  testNS('menus/info#contact'),
];

const resources = {
  [resource]: {
    [rdfx.type]: rdfx.Seq,
    [rdfx.ns('_5')]: testNS('menus/info#contact'),
    [rdfx.ns('_2')]: testNS('menus/info#governments'),
    [rdfx.ns('_0')]: testNS('menus/info#about'),
    [rdfx.ns('_3')]: testNS('menus/info#press_media'),
    [rdfx.ns('_4')]: testNS('menus/info#support'),
    [rdfx.ns('_1')]: testNS('menus/info#team'),
  },
  [testNS('menus/info#about')]: {
    [rdfx.type]: ontola.MenuItem,
    [schema.isPartOf]: testNS('menus/info'),
    [ontola.href]: rdf.namedNode('https://argu.dev/i/about'),
    [schema.name]: rdf.literal('Over Argu'),
  },
  [testNS('menus/info#team')]: {
    [rdfx.type]: ontola.MenuItem,
    [schema.isPartOf]: testNS('menus/info'),
    [ontola.href]: rdf.namedNode('https://argu.dev/i/team'),
    [schema.name]: rdf.literal('Ons team'),
  },
  [testNS('menus/info#governments')]: {
    [rdfx.type]: ontola.MenuItem,
    [schema.isPartOf]: testNS('menus/info'),
    [ontola.href]: rdf.namedNode('https://argu.dev/i/governments'),
    [schema.name]: rdf.literal('Argu voor overheden'),
  },
  [testNS('menus/info#press_media')]: {
    [rdfx.type]: ontola.MenuItem,
    [schema.isPartOf]: testNS('menus/info'),
    [ontola.href]: rdf.namedNode('https://argu.pr.co'),
    [schema.name]: rdf.literal('Pers & media'),
  },
  [testNS('menus/info#support')]: {
    [rdfx.type]: ontola.MenuItem,
    [schema.isPartOf]: testNS('menus/info'),
    [ontola.href]: rdf.namedNode('https://argu.freshdesk.com/support/home'),
    [schema.name]: rdf.literal('Help & support'),
  },
  [testNS('menus/info#contact')]: {
    [rdfx.type]: ontola.MenuItem,
    [schema.isPartOf]: testNS('menus/info'),
    [ontola.href]: rdf.namedNode('https://argu.dev/i/contact'),
    [schema.name]: rdf.literal('Contact'),
  },
};

const children = Object.keys(resources[resource]).length - 1;

describeView('Seq', components, resources, resource, () => {
  it('renders the children', async () => {
    expect((await subject).find('Seq').children()).toHaveLength(children);
  });

  describe('#sequences', () => {
    it('filters its data', async () => {
      const result = (await subject).find('Seq > ErrorBoundary > ForwardRef > Resource');
      expect(result).toHaveLength(children);
    });

    it('orders its data', async () => {
      const results = (await subject).find('Seq > ErrorBoundary > ForwardRef > Resource');

      expect(results).toHaveLength(seq.length);

      for (let i = 0; i < children; i++) {
        expect(results.at(i)).toHaveProp('subject', seq[i]);
      }
    });
  });
});
