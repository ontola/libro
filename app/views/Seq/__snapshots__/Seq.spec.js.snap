// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Seq renders a sequence fragment 1`] = `
<Seq
  dispatch={[Function]}
  label={
    NamedNode {
      "termType": "NamedNode",
      "value": "http://label.required.bugfix",
    }
  }
  subject={
    NamedNode {
      "termType": "NamedNode",
      "value": "_:g70120412320900",
    }
  }
  version="test0"
>
  <Connect(LinkedResourceContainer)
    count={6}
    data-test="Seq-0"
    first={
      NamedNode {
        "sI": 102,
        "term": "menus/info#about",
        "termType": "NamedNode",
        "value": "https://argu.dev/menus/info#about",
      }
    }
    key="<_:g70120412320900>-<https://argu.dev/menus/info#about>"
    last={
      NamedNode {
        "sI": 98,
        "term": "menus/info#contact",
        "termType": "NamedNode",
        "value": "https://argu.dev/menus/info#contact",
      }
    }
    subject={
      NamedNode {
        "sI": 102,
        "term": "menus/info#about",
        "termType": "NamedNode",
        "value": "https://argu.dev/menus/info#about",
      }
    }
  >
    <LinkedResourceContainer
      count={6}
      data-test="Seq-0"
      first={
        NamedNode {
          "sI": 102,
          "term": "menus/info#about",
          "termType": "NamedNode",
          "value": "https://argu.dev/menus/info#about",
        }
      }
      forceRender={false}
      last={
        NamedNode {
          "sI": 98,
          "term": "menus/info#contact",
          "termType": "NamedNode",
          "value": "https://argu.dev/menus/info#contact",
        }
      }
      loadLinkedObject={[Function]}
      reloadLinkedObject={[Function]}
      subject={
        NamedNode {
          "sI": 102,
          "term": "menus/info#about",
          "termType": "NamedNode",
          "value": "https://argu.dev/menus/info#about",
        }
      }
      version="test1"
    >
      <div
        className="no-view"
      >
        <p>
          We currently don"t have a view for this (&lt;https://argu.dev/menus/info#about&gt;)
        </p>
      </div>
    </LinkedResourceContainer>
  </Connect(LinkedResourceContainer)>
  <Connect(LinkedResourceContainer)
    count={6}
    data-test="Seq-1"
    first={
      NamedNode {
        "sI": 102,
        "term": "menus/info#about",
        "termType": "NamedNode",
        "value": "https://argu.dev/menus/info#about",
      }
    }
    key="<_:g70120412320900>-<https://argu.dev/menus/info#team>"
    last={
      NamedNode {
        "sI": 98,
        "term": "menus/info#contact",
        "termType": "NamedNode",
        "value": "https://argu.dev/menus/info#contact",
      }
    }
    subject={
      NamedNode {
        "sI": 108,
        "term": "menus/info#team",
        "termType": "NamedNode",
        "value": "https://argu.dev/menus/info#team",
      }
    }
  >
    <LinkedResourceContainer
      count={6}
      data-test="Seq-1"
      first={
        NamedNode {
          "sI": 102,
          "term": "menus/info#about",
          "termType": "NamedNode",
          "value": "https://argu.dev/menus/info#about",
        }
      }
      forceRender={false}
      last={
        NamedNode {
          "sI": 98,
          "term": "menus/info#contact",
          "termType": "NamedNode",
          "value": "https://argu.dev/menus/info#contact",
        }
      }
      loadLinkedObject={[Function]}
      reloadLinkedObject={[Function]}
      subject={
        NamedNode {
          "sI": 108,
          "term": "menus/info#team",
          "termType": "NamedNode",
          "value": "https://argu.dev/menus/info#team",
        }
      }
      version="test2"
    >
      <div
        className="no-view"
      >
        <p>
          We currently don"t have a view for this (&lt;https://argu.dev/menus/info#team&gt;)
        </p>
      </div>
    </LinkedResourceContainer>
  </Connect(LinkedResourceContainer)>
  <Connect(LinkedResourceContainer)
    count={6}
    data-test="Seq-2"
    first={
      NamedNode {
        "sI": 102,
        "term": "menus/info#about",
        "termType": "NamedNode",
        "value": "https://argu.dev/menus/info#about",
      }
    }
    key="<_:g70120412320900>-<https://argu.dev/menus/info#governments>"
    last={
      NamedNode {
        "sI": 98,
        "term": "menus/info#contact",
        "termType": "NamedNode",
        "value": "https://argu.dev/menus/info#contact",
      }
    }
    subject={
      NamedNode {
        "sI": 100,
        "term": "menus/info#governments",
        "termType": "NamedNode",
        "value": "https://argu.dev/menus/info#governments",
      }
    }
  >
    <LinkedResourceContainer
      count={6}
      data-test="Seq-2"
      first={
        NamedNode {
          "sI": 102,
          "term": "menus/info#about",
          "termType": "NamedNode",
          "value": "https://argu.dev/menus/info#about",
        }
      }
      forceRender={false}
      last={
        NamedNode {
          "sI": 98,
          "term": "menus/info#contact",
          "termType": "NamedNode",
          "value": "https://argu.dev/menus/info#contact",
        }
      }
      loadLinkedObject={[Function]}
      reloadLinkedObject={[Function]}
      subject={
        NamedNode {
          "sI": 100,
          "term": "menus/info#governments",
          "termType": "NamedNode",
          "value": "https://argu.dev/menus/info#governments",
        }
      }
      version="test3"
    >
      <div
        className="no-view"
      >
        <p>
          We currently don"t have a view for this (&lt;https://argu.dev/menus/info#governments&gt;)
        </p>
      </div>
    </LinkedResourceContainer>
  </Connect(LinkedResourceContainer)>
  <Connect(LinkedResourceContainer)
    count={6}
    data-test="Seq-3"
    first={
      NamedNode {
        "sI": 102,
        "term": "menus/info#about",
        "termType": "NamedNode",
        "value": "https://argu.dev/menus/info#about",
      }
    }
    key="<_:g70120412320900>-<https://argu.dev/menus/info#press_media>"
    last={
      NamedNode {
        "sI": 98,
        "term": "menus/info#contact",
        "termType": "NamedNode",
        "value": "https://argu.dev/menus/info#contact",
      }
    }
    subject={
      NamedNode {
        "sI": 104,
        "term": "menus/info#press_media",
        "termType": "NamedNode",
        "value": "https://argu.dev/menus/info#press_media",
      }
    }
  >
    <LinkedResourceContainer
      count={6}
      data-test="Seq-3"
      first={
        NamedNode {
          "sI": 102,
          "term": "menus/info#about",
          "termType": "NamedNode",
          "value": "https://argu.dev/menus/info#about",
        }
      }
      forceRender={false}
      last={
        NamedNode {
          "sI": 98,
          "term": "menus/info#contact",
          "termType": "NamedNode",
          "value": "https://argu.dev/menus/info#contact",
        }
      }
      loadLinkedObject={[Function]}
      reloadLinkedObject={[Function]}
      subject={
        NamedNode {
          "sI": 104,
          "term": "menus/info#press_media",
          "termType": "NamedNode",
          "value": "https://argu.dev/menus/info#press_media",
        }
      }
      version="test4"
    >
      <div
        className="no-view"
      >
        <p>
          We currently don"t have a view for this (&lt;https://argu.dev/menus/info#press_media&gt;)
        </p>
      </div>
    </LinkedResourceContainer>
  </Connect(LinkedResourceContainer)>
  <Connect(LinkedResourceContainer)
    count={6}
    data-test="Seq-4"
    first={
      NamedNode {
        "sI": 102,
        "term": "menus/info#about",
        "termType": "NamedNode",
        "value": "https://argu.dev/menus/info#about",
      }
    }
    key="<_:g70120412320900>-<https://argu.dev/menus/info#support>"
    last={
      NamedNode {
        "sI": 98,
        "term": "menus/info#contact",
        "termType": "NamedNode",
        "value": "https://argu.dev/menus/info#contact",
      }
    }
    subject={
      NamedNode {
        "sI": 106,
        "term": "menus/info#support",
        "termType": "NamedNode",
        "value": "https://argu.dev/menus/info#support",
      }
    }
  >
    <LinkedResourceContainer
      count={6}
      data-test="Seq-4"
      first={
        NamedNode {
          "sI": 102,
          "term": "menus/info#about",
          "termType": "NamedNode",
          "value": "https://argu.dev/menus/info#about",
        }
      }
      forceRender={false}
      last={
        NamedNode {
          "sI": 98,
          "term": "menus/info#contact",
          "termType": "NamedNode",
          "value": "https://argu.dev/menus/info#contact",
        }
      }
      loadLinkedObject={[Function]}
      reloadLinkedObject={[Function]}
      subject={
        NamedNode {
          "sI": 106,
          "term": "menus/info#support",
          "termType": "NamedNode",
          "value": "https://argu.dev/menus/info#support",
        }
      }
      version="test5"
    >
      <div
        className="no-view"
      >
        <p>
          We currently don"t have a view for this (&lt;https://argu.dev/menus/info#support&gt;)
        </p>
      </div>
    </LinkedResourceContainer>
  </Connect(LinkedResourceContainer)>
  <Connect(LinkedResourceContainer)
    count={6}
    data-test="Seq-5"
    first={
      NamedNode {
        "sI": 102,
        "term": "menus/info#about",
        "termType": "NamedNode",
        "value": "https://argu.dev/menus/info#about",
      }
    }
    key="<_:g70120412320900>-<https://argu.dev/menus/info#contact>"
    last={
      NamedNode {
        "sI": 98,
        "term": "menus/info#contact",
        "termType": "NamedNode",
        "value": "https://argu.dev/menus/info#contact",
      }
    }
    subject={
      NamedNode {
        "sI": 98,
        "term": "menus/info#contact",
        "termType": "NamedNode",
        "value": "https://argu.dev/menus/info#contact",
      }
    }
  >
    <LinkedResourceContainer
      count={6}
      data-test="Seq-5"
      first={
        NamedNode {
          "sI": 102,
          "term": "menus/info#about",
          "termType": "NamedNode",
          "value": "https://argu.dev/menus/info#about",
        }
      }
      forceRender={false}
      last={
        NamedNode {
          "sI": 98,
          "term": "menus/info#contact",
          "termType": "NamedNode",
          "value": "https://argu.dev/menus/info#contact",
        }
      }
      loadLinkedObject={[Function]}
      reloadLinkedObject={[Function]}
      subject={
        NamedNode {
          "sI": 98,
          "term": "menus/info#contact",
          "termType": "NamedNode",
          "value": "https://argu.dev/menus/info#contact",
        }
      }
      version="test6"
    >
      <div
        className="no-view"
      >
        <p>
          We currently don"t have a view for this (&lt;https://argu.dev/menus/info#contact&gt;)
        </p>
      </div>
    </LinkedResourceContainer>
  </Connect(LinkedResourceContainer)>
</Seq>
`;
